design/ceph/node-loss-rbd-rwo.md

```diff
@@ -0,0 +1,123 @@
+# Recover RBD RWO Pod in case of Node Loss
```

Comment:
if you want to cover cephfs you might need to update the title and the filename.

Reply:
updated